contracts/governance/Vesting/fouryear/FourYearVestingStorage.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.5.17;
+
+import "../../../openzeppelin/Ownable.sol";
+import "../../../interfaces/IERC20.sol";
+import "../../Staking/Staking.sol";
+import "../../IFeeSharingProxy.sol";
+import "../../../openzeppelin/Address.sol";
+
+/**
+ * @title Four Year Vesting Storage Contract.
+ *
+ * @notice This contract is just the storage required for four year vesting.
+ * It is parent of FourYearVestingLogic and FourYearVesting.
+ *
+ * @dev Use Ownable as a parent to align storage structure for Logic and Proxy contracts.
+ * */
+contract FourYearVestingStorage is Ownable {
+	/// @notice The SOV token contract.
+	IERC20 public SOV;
+
+	/// @notice The staking contract address.
+	Staking public staking;
+
+	/// @notice The owner of the vested tokens.
+	address public tokenOwner;
+
+	/// @notice Fee sharing Proxy.
+	IFeeSharingProxy public feeSharingProxy;
+
+	/// @notice The cliff. After this time period the tokens begin to unlock.
+	uint256 public constant cliff = 4 weeks;
```

Comment:
Please use upper case for all constants.

Reply:
Done